src/protocol-data-converter.ts

```diff
@@ -20,20 +20,22 @@ function protocolDataToIlpAndCustom ({ protocolData }) {
 
   return {
     protocolMap,
-    ilp: protocolMap.ilp,
-    custom: protocolMap.custom
+    ilp: protocolMap['ilp'],
+    custom: protocolMap['custom']
   }
 }
 
-function ilpAndCustomToProtocolData ({ ilp, custom, protocolMap }) {
+export function ilpAndCustomToProtocolData (data: { ilp?: Buffer, custom?: Object , protocolMap?: Map<string, Buffer | string | Object> }): Array<BtpSubProtocol> {
```

Comment:
Here, it makes more sense to wrap the three arguments into one "data" argument. https://stackoverflow.com/questions/12826977/multiple-arguments-vs-options-object suggest doing this if there are more than four arguments, I guess it's a matter of taste, but maybe we should add a rule about this to our in-house style guide :)

Reply:
This is done as is fro backwards compatibility